sdcm/provision/scylla_yaml/scylla_yaml.py

```diff
@@ -58,6 +58,7 @@ def set_endpoint_snitch(cls, endpoint_snitch: str):
     rpc_interface: str = "eth1"
     rpc_interface_prefer_ipv6: bool = False
     seed_provider: List[SeedProvider] = [SeedProvider(class_name='org.apache.cassandra.locator.SimpleSeedProvider')]
+    consistent_cluster_management: bool = False
```

Comment:
for now, is it the default in `scylla.yaml`?

Reply:
It's the default in our `ScyllaYaml` class, since it would refuse unrecognized options being added.

Reply:
ScyllaYaml doesn't support seeing value to None ?

i.e. it shouldn't appear by default

Since the default in scylla might change in following versions, I don't want SCT to mask that